pkg/reconciler/events/scheduler/scheduler_test.go

```diff
@@ -1266,8 +1266,8 @@ func TestAllCases(t *testing.T) {
 					reconcilertestingv1.WithCloudSchedulerSourceSchedule(onceAMinuteSchedule),
 					reconcilertestingv1.WithInitCloudSchedulerSourceConditions,
 					reconcilertestingv1.WithCloudSchedulerSourceJobReady(jobName),
-					reconcilertestingv1.WithCloudSchedulerSourceTopicFailed("TopicDeleted", fmt.Sprintf("Successfully deleted Topic: %s", schedulerName)),
-					reconcilertestingv1.WithCloudSchedulerSourcePullSubscriptionFailed("PullSubscriptionDeleted", fmt.Sprintf("Successfully deleted PullSubscription: %s", schedulerName)),
+					reconcilertestingv1.WithCloudSchedulerSourceTopicReady("", ""),
```

Comment:
Should we write something like `reconcilertestingv1.WithCloudSchedulerSourceTopicReady(testTopicID, testProject)`?

Reply:
No, because https://github.com/google/knative-gcp/blob/master/pkg/reconciler/intevents/reconciler.go#L271 it sets topicID and projectID to be empty when deleting the topic.

Reply:
In this case, instead of using "WithCloudSchedulerSourceTopicReady", maybe write a new method like `WithCloudSchedulerSourceTopicDeleted()`which makes the topicID to be "".
`WithCloudSchedulerSourceTopicReady()` is kind of confusing.

Reply:
Does that mean methods for `WithCloudSchedulerSourcePullSubscriptionDeleted()` is also needed? Otherwise it will be more confusing that the topic is deleted but pullSubscription is ready? I think it kind of going back to the question that my initial thought of marking status as not Ready when deletion happens successfully because I was trying to avoid this kind of confusion.

Reply:
Yep. We will also need WithCloudSchedulerSourcePullSubscriptionDeleted().
I think this is a little bit different to the issue of marking status as NotReady when deletion happens successfully.
My point here is to make it explicitly that the resources get deleted but here the status is still ready.

Reply:
Okay. Sounds good. So basically the `WithCloudSchedulerSourcePullSubscriptionDeleted()` and `WithCloudSchedulerSourceTopicDeleted()` are just wrapper but inside it's still markStatus as Ready.

Reply:
Yep.  But maybe in `WithCloudSchedulerSourcePullSubscriptionDeleted()`, write some comments to explain why we have this wrapper.

Reply:
Will do. Yes. Because this wrapper and `WithCloudSchedulerSourcePullSubscriptionReady()` have the same implementation.